frontend/package.json

```diff
@@ -23,20 +23,27 @@
     "typescript": "^3.8.2"
   },
   "scripts": {
-    "start": "parcel serve index.html --port 2345 --hmr-port 2346 --no-source-maps",
-    "build": "rm dist/* -rf; parcel build index.html --no-source-maps",
+    "start": "parcel serve index.html --port 2345 --hmr-port 2346",
+    "build": "rm dist/*; parcel build index.html",
     "test": "jest"
   },
   "dependencies": {
+    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
     "@material-ui/core": "^4.10.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.50",
+    "@parcel/transformer-react-refresh-babel": "^2.0.0-alpha.3",
+    "@parcel/watcher": "^2.0.0-alpha.8",
```

Comment:
Nor this

Reply:
same here, as in my previous comment, will check it

Reply:
solved